ca/certificate-authority_test.go

```diff
@@ -96,6 +96,24 @@ var (
 	// Edited signature to become invalid.
 	WrongSignatureCSR = mustRead("./testdata/invalid_signature.der.csr")
 
+	// CSR generated by Go:
+	// * Random public key
+	// * CN = not-example.com
+	// * Includes an extensionRequest attribute for a well-formed TLS Feature extension
+	MustStapleCSR = mustRead("./testdata/must_staple.der.csr")
+
+	// CSR generated by Go:
+	// * Random public key
+	// * CN = not-example.com
+	// * Includes an extensionRequest attribute for an empty TLS Feature extension
+	TLSFeatureUnknownCSR = mustRead("./testdata/tls_feature_unknown.der.csr")
```

Comment:
The test case you're asking for already exists -- that's what `TLSFeatureUnknownCSR` is.